lib/modules/platform/bitbucket/index.ts

```diff
@@ -621,9 +621,22 @@ export async function addReviewers(
   // TODO #7154
   const { title } = (await getPr(prId))!;
 
+  const UUIDRegex = regEx(
+    /^\{[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\}$/,
+    'i'
+  );
```

Comment:
looks like a good candidate to extract a util function. not sure if it should be in regex or string utils.

@rarkins @secustor WDYT?

Reply:
Maybe even to [a UUID type][1] too :upside_down_face:

/s

[1]: https://ybogomolov.me/type-level-uuid/

Reply:
Do you use somewhere else UUIDs with additional functionality? If yes, we could create a own utils module else, I would put it with the regex utils.

Reply:
I checked around and didn't find any other UUID parsing. I put it in the regex util.